src/webvr-manager.js

```diff
@@ -42,6 +42,10 @@ var Wakelock = require('./wakelock.js');
 function WebVRManager(renderer, effect, params) {
   this.params = params || {};
 
+  // Give a unique to ID to each manager.
+  this.prefix = 'webvr';
```

Comment:
The only reason was not to start an ID or className with a number. Could be optional.